storops/lib/tasks.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# Copyright (c) 2016 EMC Corporation.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+import pqueue
+import threading
+import time
+from storops.exception import StoropsException
+
+__author__ = 'Peter Wang'
+LOG = logging.getLogger(__name__)
+
+
+class PQueue(object):
+    DEFAULT_INTERVAL = 300
+
+    def __init__(self, path, interval=None):
+        self.path = path
+        self._q = pqueue.Queue(self.path)
+        self._interval = interval if interval else self.DEFAULT_INTERVAL
```

Comment:
Use `interval is None` instead of `if interval`.  What if I want to set interval to 0?